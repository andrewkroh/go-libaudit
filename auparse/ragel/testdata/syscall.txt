type=SYSCALL msg=audit(1490995752.404:20614537): arch=c000003e syscall=44 success=yes exit=464 a0=9 a1=7f56476614b0 a2=1d0 a3=0 items=0 ppid=1240 pid=1287 auid=4294967295 uid=0 gid=0 euid=0 suid=0 fsuid=0 egid=0 sgid=0 fsgid=0 tty=(none) ses=4294967295 comm="charon" exe=2F7573722F6C6962657865632F7374726F6E677377616E2F636861726F6E202864656C6574656429 key=(null)