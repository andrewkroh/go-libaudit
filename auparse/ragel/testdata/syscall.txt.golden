type: SYSCALL
typeid: 0
timestamp: "1490995752.404"
sequence: "20614537"
values:
  a0: NUM:9
  a1: 7f56476614b0
  a2: 1d0
  a3: NUM:0
  arch: c000003e
  auid: NUM:4294967295
  comm: charon
  egid: NUM:0
  euid: NUM:0
  exe: /usr/libexec/strongswan/charon (deleted)
  exit: NUM:464
  fsgid: NUM:0
  fsuid: NUM:0
  gid: NUM:0
  items: NUM:0
  key: (null)
  pid: NUM:1287
  ppid: NUM:1240
  ses: NUM:4294967295
  sgid: NUM:0
  success: "yes"
  suid: NUM:0
  syscall: NUM:44
  tty: (none)
  uid: NUM:0
message: ""
original: 'type=SYSCALL msg=audit(1490995752.404:20614537): arch=c000003e syscall=44 success=yes exit=464 a0=9 a1=7f56476614b0 a2=1d0 a3=0 items=0 ppid=1240 pid=1287 auid=4294967295 uid=0 gid=0 euid=0 suid=0 fsuid=0 egid=0 sgid=0 fsgid=0 tty=(none) ses=4294967295 comm="charon" exe=2F7573722F6C6962657865632F7374726F6E677377616E2F636861726F6E202864656C6574656429 key=(null)'
